scss/core/abstracts/_colors.scss

```diff
@@ -358,3 +358,16 @@ $oranges: map-merge((
   "800": $orange-800,
   "900": $orange-900
 ), $oranges);
+
+$grid-shadow-color: rgba($gray-700,0.1);
+$text-gray: $gray-700;
+$healthy-primary : $green-500;
+$healthy-secondary :$green-300;
+$unhealthy-primary : $red-500;
+$unhealthy-secondary : $red-300;
+$link-color : $blue-500;
+$btn-github:$gray-800;
+$code-wrap-bg:$gray-000;
+$active-code-block-shadow:$violet-400;
+$font-color-gray:$gray-700;
+$bg-g:rgba($gray-000, 0.5);
```

Comment:
colors.scss only includes in the de-code color palette. It's should include other color mappings. Color variable for a comment should go in variable.scss

Reply:
Done